tests/unit/test-class-wc-payments-express-checkout-button-display-handler.php

```diff
@@ -146,7 +154,8 @@ private function make_wcpay_gateway() {
 			$mock_action_scheduler_service,
 			$mock_rate_limiter,
 			$mock_order_service,
-			$mock_dpps
+			$mock_dpps,
+			$this->mock_localization_service
```

Comment:
Here we don't need a new property, and we can dispatch `$this->createMock( WC_Payments_Localization_Service::class )` right here.

Reply:
Done https://github.com/Automattic/woocommerce-payments/pull/6920/commits/d72b58abc92d3ef08193fa5d1b35ee5533055f60